manager/runner.go

```diff
@@ -513,6 +534,12 @@ func (r *Runner) Run() error {
 				// Record that at least one template was rendered.
 				renderedAny = true
 
+				// Send the signal that the template got rendered
+				select {
+				case r.renderedCh <- tmpl.Path:
+				default:
```

Comment:
We don't want consul-template to be blocked by the listener. Though I had a decent idea. We should add a map[string]time.Time and expose it. So that if the caller doesn't keep up they can get the map and see what they have missed